### ISSUE TITLE
Add NQ like graphic obituaries

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#154]
Author: @IR4T4 (IR4T4)
Date: 2012-12-18
Redmine Issue: 154, https://dev.etlegacy.com/issues/154

---

.. text obituaries eat half of the screen ... let's fix that.

---

Related commit(s):
0bc269eeb97931ab9b64e802d29e1f7f5b114cbc - cgame: additional param color for CG_AddPMItem() t..
 6a0ccad38daed900ed67961c2343ebf61b013884 - game: send weapon in obituaries events refs 154
 bf91fb97841a365986bf211c343717a4839a79ba - cgame: added cg_graphicObituaries, refs 154
 935b72dbe7769098ae7815cf6f6e481deb56153d - ui: added cg_graphicObituaries option, refs 154
 414ac5ed7e6796da006a8a278f3e07ebfea4f545 - cgame: minor cleanup, refs 154... and compile f..
 09b10e063c65e96e412637948e523e8fd506bebf - game: MOD and splashMOD fixed for allied grenade r..
 46e0c2d57007e29c28d39d597d8788829330d678 - cgame: fixed cg_obituaries 2 display order, refs ..
 2597bccb8b56a5373410e0190957ffc742c89786 - ui: added cg_graphicObituaries 2 option, refs 154
 d9a17c34bfe08c2de51345ed4d33b279c9b95345 - cgame: fixed teamkill colored console message, ref..

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2013-11-04T21:12:06Z

---

This is mostly done.

However, I've seen that the allies nade (pineapple nade) is displayed with the axis nade icon. This is not strictly related to this feature, but might be related to weapon tables.

There is also a new warning to fix (warning: implicit declaration of function ‘weaponIconScale’).

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2013-11-06T19:51:41Z

---

Fixed some regression and enabled 'cg_graphicObituaries = 2' in the UI.

I played some games and checked for other icon errors, but couldn't find any. Some weapons use the generic skull icon and might be improved later (artillery strike, mounted MG, tank MG) with the planned weapon table, but that is not directly related to this feature so I'm closing it.
